### PR TITLE
Updated the main bundles article for Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**
I've removed the `Bundle Directory Structure` contents ... but I plan to move them to other bundle articles in a separate PR.

### CODE REVIEW

File: bundles.rst [hunk 1]
```diff
@@ -6,179 +6,52 @@
...
-        }
-
-        return $bundles;
-    }
-
-With the ``registerBundles()`` method, you have total control over which bundles
-are used by your application (including the core Symfony bundles).
+.. caution::
+
+    In Symfony versions prior to 4.0, it was recommended to organize your own
```
Comment (xabbuh) [inline bundles.rst]: I do not really agree with this sentence. At least, it is a bit confusing as we didn't recommend to use bundles, but there only was a single AppBundle. I think that's not really clear right now.

Comment (javiereguiluz) [reply bundles.rst]: We'll need more opinions here, but I thought this:

* Symfony 2: organize your own code in bundles (1 bundle per feature: UserBundle, ProuctBundle, ...)
* Symfony 3: organize your own code in bundles (just 1 bundle per app: AppBundle)

Saying: *"in prior Sf versions we recommended you to organize your own code in bundles"* is compatible with that (in my opinion).

Comment (wouterj) [reply bundles.rst]: I agree with @javiereguiluz here. "organize in bundles" doesn't have to refer to multiple bundles

File: bundles.rst [hunk 2]
```diff
@@ -6,179 +6,52 @@
...
+    application code using bundles. This is no longer recommended and bundles
+    should only be used as third-party plugins that add features in your
+    applications.
```
Comment (wouterj) [inline bundles.rst]: "third-party" refers to another party, but it's perfectly fine to use a bundle made by your own company. What about:

 > This is no longer recommended and bundles should only be used to share code and features between multiple applications.

**Review by wouterj — APPROVED** [review]
An article describing how to create a bundle is now badly needed, but I agree with the changes in this PR!

File: bundles.rst [hunk 3]
```diff
@@ -6,179 +6,52 @@
...
 
-.. toctree::
-    :maxdepth: 1
-    :glob:
-
-    bundles/*
+* :doc:`/bundles/installation`
+* :doc:`/bundles/remove`
+* :doc:`/bundles/override`
+* :doc:`/bundles/inheritance`
```
Comment (weaverryan) [inline bundles.rst]: Let's stop linking to this - it'll still be findable by Google.

File: bundles.rst [hunk 4]
```diff
@@ -6,179 +6,52 @@
...
 
 Learn more
 ----------
 
-.. toctree::
-    :maxdepth: 1
-    :glob:
-
-    bundles/*
+* :doc:`/bundles/installation`
```
Comment (weaverryan) [inline bundles.rst]: And let's remove this link - I'm going to open a PR to remove this entirely

**Review by weaverryan — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by wouterj** [comment]
Woohooo, just another article updated!

**Pull request closed without merging**